src/resources/basis.js

```diff
@@ -475,11 +476,15 @@ function basisDownloadFromConfig(callback) {
                           urlBase + wasmModule.wasmUrl,
                           urlBase + wasmModule.fallbackUrl,
                           callback);
-        } else {
-            // #ifdef DEBUG
-            console.warn("WARNING: unable to load basis wasm module - no config was specified");
-            // #endif
+
+            return true;
         }
+
+        // #ifdef DEBUG
+        console.warn("WARNING: unable to load basis wasm module - no config was specified");
+        // #endif
+
+        return false;
```

Comment:
The first block isn't returning true so I'd suggest:
```suggestion
        } else {
            // #ifdef DEBUG
            console.warn("WARNING: unable to load basis wasm module - no config was specified");
            // #endif
            return false;
        }
        return true;
```